test/response/api/v1/attendances/get.json

```diff
@@ -1,337 +1,504 @@
 {
   "items": [
     {
+      "id": 41761,
+      "team": {
+        "id": 3,
+        "href": "http://amam.sample.com/api/v1/teams/3"
+      },
+      "member": {
+        "id": "100606116",
+        "href": "http://amam.sample.com/api/v1/members/100606116"
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "http://amam.sample.com/api/v1/attendance_types/5"
```

Comment:
[Underscores vs. hyphens in URLs](http://ux.stackexchange.com/questions/9303/underscores-vs-hyphens-in-urls)
[Google](https://support.google.com/webmasters/answer/76329) recommended to use hyphenated URLs.

But for famous public APIs (Facebook, Netflix, etc..) seem using underscored URLs.
[Hyphen, underscore, or camelCase as word delimiter in URIs?](http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)

What do you think? @sideroad @nabuchi @taka66